### ISSUE TITLE
libobsd doesn't work with non-existant headers

### ISSUE THREAD

**Issue by neheb** [guijan/libobsd#1]
compiling libtirpc with musl (which has no sys/queue.h) errors with:

```
../src/clnt_bcast.c:43:10: fatal error: sys/queue.h: No such file or directory
   43 | #include <sys/queue.h>
```

**Comment by guijan** [comment]
libtirpc's _configure.ac_ doesn't try to use libbsd or libobsd if the operating system is missing the header. It should work if the right lines are added to their build definitions, that would look a bit like this: https://github.com/resurrecting-open-source-projects/scrot/blob/ed96aa4ae3d69c186b3dda88543cc541acaf83c2/configure.ac#L33-L43

**Comment by neheb** [comment]
my attempt was actually with a meson.build file with dependency('libbsd-overlay') and wrap_mode=forcefallback.

**Comment by guijan** [comment]
Fixed by 495ca2565c4abdb67e289ea40a62956c19eef3d2.
